test/system/005-info.bats

```diff
@@ -37,7 +37,7 @@ cgroupVersion: v[12]
     # FIXME: if we're ever able to get package versions on Debian,
     #        add '-[0-9]' to all '*.package' queries below.
     tests="
-host.buildahVersion       | [0-9.]
+host.buildahVersion       | [0-9.]\\\+
```

Comment:
This will now fail if we ever vendor a `-dev` buildah, or something with letters.

Reply:
We regularly do this during development, will this block a merge to upstream?

Reply:
It probably would. Let me think of how to handle that; I will pore through git logs, find patterns, and resubmit. Thanks for letting me know.